content/general/information-technology/information-technology.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Information Technology'
+Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cybersecurity'
+CatalogContent:
+  - 'introduction-to-information-technology'
+---
```

Comment:
You still need to format the Metadata as a term entry, it's still laid out as a topic.